PyRoute/Calculation/TradeCalculation.py

```diff
@@ -228,26 +235,54 @@ def route_update_simple(self, route):
         """
         distance = self.route_distance(route)
 
+        source = route[0]
+        target = route[-1]
+
         # Internal statistics
-        self.galaxy.ranges[route[0]][route[-1]]['actual distance'] = distance
-        self.galaxy.ranges[route[0]][route[-1]]['jumps'] = len(route) - 1
+        rangedata = self.galaxy.ranges[source][target]
+        rangedata['actual distance'] = distance
+        rangedata['jumps'] = len(route) - 1
+
+        if False and source.is_landmark != target.is_landmark:
+            if source.is_landmark:
+                # landmark = source
+                other = target
+            else:
+                # landmark = target
+                other = source
+
+            items = self.galaxy.distance_alt_tracking.items()
+            #combos = [(k, v) for (k, v) in items if 4 < abs(v - distance)]
+
+            for item in items:
+                straight = other.hex_distance(item[0])
+                bound = abs(distance - item[1])
+                if bound > straight:
+                    self.galaxy.landmarks[(other, item[0])] = bound
```

Comment:
I don't see where you add the landmarks dict to the `Galaxy.__init__()`. It's not required by python, but does improve readability of the code.

Reply:
Resolve this?